src/core/viz/expressions/aggregation/globalAggregation.js

```diff
@@ -177,7 +187,9 @@ function generateGlobalAggregattion(metadataPropertyName) {
 /**
  * Return the Nth percentile of the feature property for the entire source data.
  *
- * @param {carto.expressions.prop} property - property expression of date or number type
+ * Note: `globalSum` input property can only be a {@link carto.expressions.prop|carto.expressions.prop}, not other expressions.
+ *
+ * @param {Number} property - property expression of number type
```

Comment:
The second parameter is missing :slightly_frowning_face: Sorry, we should have caught this kind of things in other places, not in your PR.

Reply:
It's OK, let's fix it.

Reply:
This is still needed (github marked it as outdated)